raiden_contracts/constants.py

```diff
@@ -8,7 +8,7 @@
 from raiden_contracts.utils.type_aliases import ChainID
 
 # Do not change this, this is handled by bumpversion with .bumpversion_contracts.cfg
-CONTRACTS_VERSION = "0.20.0"
+CONTRACTS_VERSION = "0.21.0"
```

Comment:
This constant is used in the client currently to find the newest contracts_version.